WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3270,6 +3270,7 @@ private boolean isRemoteAutoSaveError() {
     private PostModel handleRemoteAutoSave(boolean isError, PostModel post) {
         // We are in the process of remote previewing a post from the editor
         if (!isError && isUploadingPostForPreview()) {
+            mViewModel.hideSavingDialog();
```

Comment:
I'm wondering - don't we need to hide the dialog even when the remote-autosave fails? Or is that handled in a different code path?

Reply:
I've moved the hideSavingDialog out of the if condition. I don't think all the paths are covered. I'll ping the people responsible for this logic